test_requirements.txt

```diff
@@ -3,3 +3,4 @@ pytest-tornasync
 pytest-console-scripts
 flake8
 hacking
+requests-unixsocket
```

Comment:
This can be removed now that Notebook 6.1.1 is available and addresses this issue.